vss-extension.json

```diff
@@ -63,12 +59,11 @@
         "type": "git",
         "uri": "https://github.com/pulumi/pulumi-az-pipelines-task"
     },
+    "scopes": ["vso.code_write"],
```

Comment:
Missed adding a callout for this change when I opened the PR. This scope is needed so that the build gets an access token with this permission. Note that by specifying a scope we are effectively losing access to the default scopes that this task had but never used.

According to the docs:

> If no scopes are specified, extensions are only provided access to user profile and extension data.

Ref: https://docs.microsoft.com/en-us/azure/devops/extend/develop/manifest?view=azure-devops#scopes